regression/cbmc-incr-smt2/nondeterministic-int-assert/incremental_solver_called.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+empty.c
+--incremental-smt2-solver z3
+WARNING: Leaving variables of static lifetime uninitialised.
```

Comment:
I don't think this warning is part of this test.

Reply:
Ok. I'll remove this one, as it is probably throwing the net a little too wide.

Reply:
Great. I am hoping this test will survive a lot of the PRs without changes as a very basic sanity test. If we only test the string of the command then we can keep it when we reinstate the CPROVER initialization.

Reply:
This is now gone.